backend/init.py

```diff
@@ -0,0 +1,2 @@
+from main import db
```

Comment:
nothing. but [python style guide](https://www.python.org/dev/peps/pep-0008/) says to add 2 blank spaces between imports and the code.

you can run [flake8](http://flake8.pycqa.org/en/latest/) and/or [pep8](https://pypi.python.org/pypi/pep8) to see the suggestions/corrections and make the code more pythonic.